src/components/screens/WalletKit/DappConnectionBottomSheetContent.tsx

```diff
@@ -60,89 +76,186 @@ const DappConnectionBottomSheetContent: React.FC<
         "user_rejected",
       );
     }
+
     onCancel();
   };
 
-  return (
-    <View className="flex-1 justify-center items-center mt-2">
-      <App
-        size="lg"
-        appName={dappMetadata.name}
-        favicon={dappMetadata.icons[0]}
-      />
-      <View className="mt-4" />
-      <Text lg primary medium style={{ textAlign: "center" }}>
-        {dappMetadata.name}
-      </Text>
-      <View className="mt-1" />
-      {dappDomain && (
-        <Text sm secondary>
-          {dappDomain}
-        </Text>
-      )}
-      <View className="mt-2" />
-      <Badge
-        variant="secondary"
-        size="md"
-        icon={<Icon.Link01 size={16} />}
-        iconPosition={IconPosition.LEFT}
-      >
-        {t("dappConnectionBottomSheetContent.connectionRequest")}
-      </Badge>
-      <View className="flex-row items-center mt-6 p-6 bg-background-tertiary rounded-xl justify-center">
-        <Text md secondary style={{ textAlign: "center" }}>
-          {t("dappConnectionBottomSheetContent.disclaimer")}
-        </Text>
-      </View>
-      <View className="w-full flex-row items-center mt-6 px-6 py-4 bg-background-primary border border-border-primary rounded-xl justify-between">
-        <View className="flex-row items-center">
-          <Icon.UserCircle size={16} color={themeColors.foreground.primary} />
-          <Text
-            md
-            secondary
-            style={{ textAlign: "center", marginLeft: pxValue(8) }}
-          >
-            {t("wallet")}
-          </Text>
-        </View>
-        <View className="flex-row items-center">
-          <Text
-            md
-            secondary
-            style={{ textAlign: "center", marginRight: pxValue(8) }}
-          >
-            {account?.accountName}
-          </Text>
+  const listItems = [
+    {
+      icon: <Icon.Wallet01 size={16} color={themeColors.foreground.primary} />,
+      title: t("wallet"),
+      trailingContent: (
+        <View className="flex-row items-center gap-2">
           <Avatar
             size="sm"
-            hasBorder={false}
             publicAddress={account?.publicKey ?? ""}
+            hasBorder={false}
+            hasBackground={false}
           />
+          <Text md primary>
+            {account?.accountName}
+          </Text>
         </View>
+      ),
+      titleColor: themeColors.text.secondary,
+    },
+    {
+      icon: <Icon.Globe01 size={16} color={themeColors.foreground.primary} />,
+      title: t("network"),
+      trailingContent: (
+        <Text md primary>
+          {network === NETWORKS.PUBLIC ? t("mainnet") : t("testnet")}
```

Comment:
I think the networks name here should be consistent with what we are displaying on Settings/Network regardless of how it's being displayed on Figma right now. Which would mean "Main Net" and "Test Net".